docs/getting-started/homepage.md

```diff
@@ -0,0 +1,166 @@
+
+At the end of this tutorial, you can expect:
+
+- Your Backstage app to have a dedicated homepage instead of Software Catalog.
+- Understand the composability of homepage and how to start customizing it for your own organization.
+
+### Prerequisites
+
+Before we begin, make sure
+
+- You have created your own standalone Backstage app using [`@backstage/create-app`](index.md#create-your-backstage-app) and not using a fork of the [backstage](https://github.com/backstage/backstage) repository.
+- You do not have an existing homepage, and by default you are redirected to Software Catalog when you open Backstage.
+
+Now, let's get started by installing the home plugin and creating a simple homepage for your Backstage app.
+
+### Setup homepage
+
+#### 1. Install the plugin
+
+```
+# From your Backstage application directory
+cd packages/app
+yarn add @backstage/plugin-home
+```
+
+#### 2. Create a new HomePage component
+
+Inside your `packages/app` directory, create a new file where our new homepage component is going to live. Create `packages/app/src/components/home/HomePage.tsx` with the following initial code
+
+```tsx
+import React from 'react';
+
+export const HomePage = () => {
+  /* We will shortly compose a pretty homepage here. */
+  return <h1>Welcome to Backstage!</h1>;
+};
+```
+
+#### 3. Update router for the root `/` route
+
+If you don't have a homepage already, most likely you have a redirect setup to use the Catalog homepage as a homepage.
+
+Inside your `packages/app/src/App.tsx`, look for
+
+```tsx
+const routes = (
+  <FlatRoutes>
+    <Navigate key="/" to="catalog" />
+```
+
+Let's replace the `<Navigate>` line and use the new component we created in the previous step as the new homepage.
+
+```diff
+// File: packages/app/src/App.tsx
+
++ import { HomepageCompositionRoot } from '@backstage/plugin-home';
++ import { HomePage } from './components/home/HomePage';
+
+// ...
+const routes = (
+  <FlatRoutes>
+-   <Navigate key="/" to="catalog" />
++   <Route path="/" element={<HomepageCompositionRoot />}>
++     <HomePage />
++   </Route>
+// ...
+
+)
+```
+
+### 4. Update sidebar items
+
+Let's update the route for "Home" in the Backstage sidebar to point to the new homepage. We'll also add a Sidebar item to quickly open Catalog.
+
+<table>
+  <tr>
+    <th><img data-zoomable src="../assets/getting-started/sidebar-without-catalog.png" alt="Sidebar without Catalog" /></td>
+    <th><img data-zoomable src="../assets/getting-started/sidebar-with-catalog.png" alt="Sidebar with Catalog" /></td>
+  </tr>
+  <tr align="center">
+    <td>Before</td>
+    <td>After</td>
+  </tr>
+</table>
+
+The code for the Backstage sidebar is most likely inside your [`packages/app/src/components/Root/Root.tsx`](https://github.com/backstage/backstage/blob/master/packages/app/src/components/Root/Root.tsx).
+
+Let's make the following changes
+
+```diff
+// Other imports
++ import CategoryIcon from '@material-ui/icons/Category';
```

Comment:
I noticed recently there is a `CatalogIcon` exported from `core-components` that might be more appropriate to use here.

That said, there are a few other system icons (eg. `DocsIcon`) that could be used in the demo app and app template as well.

Reply:
Good point @andrewthauer ! I couldn't find the CatalogIcon in the storybook so I didn't know. Maybe a more general topic on Icons consistency 😁

Reply:
cc @katz95 and @emmaindal about the icons 🙏